docs/modules/user-guide/partials/proc_adding-a-volume-component-to-a-devfile.adoc

```diff
@@ -0,0 +1,41 @@
+[id="proc_adding-a-volume-component-to-a-devfile_{context}"]
+= Adding a volume component to a devfile
+
+[role="_abstract"]
+You can use a volume component to share files among container components and collaborate easily with other teams during the development process.
+
+.Prerequisites
+
+* xref:adding-schema-version-to-a-devfile.adoc[]
+* xref:adding-a-name-to-a-devfile.adoc[]
+* xref:adding-components-to-a-devfile.adoc[]
+
+.Procedure
+
+. Define a `volume` component in your devfile and specify a volume size for it.
```

Comment:
Here, we'd want to keep it component since that's the terminology in the yaml. However, you're right in our other docs between definitively selecting `property` or `type` based off the context.